File: codewars/7kyu/shortestWord.py
"""
Simple, given a string of words, return the length of the shortest word(s).

String will never be empty and you do not need to account for different data types.
"""

def shortest_word(string):
    listed = string.split(" ")
    shortest_word = min(listed, key=len)
    return len(shortest_word)

### OTHER SOLUTIONS ###

def find_short(s):
    return min(len(x) for x in s.split())

File: codewars/7kyu/shortestWord.js
/*
Simple, given a string of words, return the length of the shortest word.

String will never be empty and you do not need to account for different data types.
*/

const shortestWord = (stringOfWords) => {
	const words = stringOfWords.split(" ");
	const wordsLengths = words.map(word => word.length);
	return Math.min(...wordsLengths);
}
